code/game/mecha/mecha.dm

```diff
@@ -949,8 +954,7 @@
 			return mmi_moved_inside(mmi_as_oc, user)
 		else
 			to_chat(user, "<span class='warning'>Occupant detected!</span>")
-	else

```

Comment:
Why did this else get deleted?

Reply:
it was not needed, I'd have to reference the entire proc to know for sure - but I'd say the reasoning was the "Protip: Early Returns are preferred" reference in the wiki.

Reply:
Actually your right - I fixed it  in the latest commit; I'm pretty sleepless..